SWIG/calendars.i

```diff
@@ -113,10 +113,10 @@ class Calendar {
             return self->name()+" calendar";
         }
         #if defined(SWIGPYTHON) || defined(SWIGJAVA)
-        bool __eq__(const Calendar& other) {
+        bool operator==(const Calendar& other) {
```

Comment:
Since we're using the C++ name, this could go as a simple declaration outside the `%extend`, right?

Reply:
No, wait, it's a friend function in C++, I'm not sure if that works with SWIG...